### ISSUE TITLE
The ability to block certain servers to the client on a schedule (by time)

### ISSUE THREAD

**Issue by fizsef** [AdguardTeam/AdGuardHome#5870]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to request a feature or enhancement and not ask a question

### Description

Dear admins. The option of using a schedule by time is very lacking in the user filtering rules. For example, you need to block access to social networks to a certain client on a schedule. Please implement this functionality in new versions.

**Comment by theschles** [comment]
FYI I have the same need:
* I have two sons with school-issued Chromebooks -- so I can't install Qustodio or any other blocker on them
* I'd like Youtube to be blocked on the Chromebooks by default
* If either son has done his homework and practiced his musical instrument, I'd like to be able to unblock for a designated period of time Youtube as a reward -- but only that particular son's computer
* ...and then after that designated period of time is up, have Youtube automatically reblocked on that son's computer

Right now, the only way I can see doing this -- with user-friendliness that my non-techie wife would be happy using -- would be:
* In AdGuard Home: 
  * Create separate "Settings > Client settings" entries for each son, making sure:
    * In the "Settings" tab when editing the "Client" entry: 
      * "Block domains using filters and host files" checkbox is enabled
    * In the "Block Specific Services" tab when editing the "Client" entry:
      * Disable "Use global blocked services"
      * Enable "Youtube"
    * ...now save the Client entry
  * For Chromebooks: 
    * In "Filters > Custom filtering rules":
      * Add: `@@||accounts.youtube.com^$important` and click the Apply button
        * Why? Chromebook authentication uses `accounts.yahoo.com`, there needs to be a custom filter rule to make sure it is NEVER blocked

* In Home Assistant:  
  * (FYI I haven't figured out specifically how to do this yet....will update this issue when I have figured it out)
  * Create buttons "Reward Son X with Youtube" and "Reward Son Y with Youtube"
  * Configure each button to call a particular script -- or to call a single script with a parameter of the name of the son -- which will:
    * Issue a REST call to AdGuard Home to unblock Youtube for that particular son's Client entry
    * Insert a Cron job to re-block Youtube in an hour (or whatever designated period of time)

**Comment by schzhn** [comment]
@fizsef, this feature is available since [v0.107.37](https://github.com/AdguardTeam/AdGuardHome/releases/tag/v0.107.37).

**Comment by ainar-g** [comment]
@schzhn, that one is about Blocked Services.  This one is about filtering-rule lists, as far as I can tell.

**Comment by fizsef** [comment]
> @schzhn, that one is about Blocked Services. This one is about filtering-rule lists, as far as I can tell.

Yes, it's about the rules. To be able to set an action schedule in each rule.
